cpp/include/cugraph/algorithms.hpp

```diff
@@ -24,12 +24,135 @@
 #include <cugraph/internals.hpp>
 #include <cugraph/legacy/graph.hpp>
 
+#include <cugraph-ops/aggregator.hpp>
 #include <cugraph-ops/graph/sampling.hpp>
```

Comment:
@ChuckHastings @kaatish Should we better move cugraph-ops interface functions to a separate file? (Or even a separate directory?)

Reply:
I mean the declarations for `aggregate_forward`, `aggregate_concatenate_forward`, `aggregate_backward` and `aggregate_concatenate_backward` may better go to a separate file?

Reply:
I would say at least a separate file.  We probably need to settle on how exactly to package these functions before deciding on whether it goes in this directory or another one.  But if we put it in a separate file, it's easy to move the file to a different directory if we decide that needs to happen.